packages/store/src/symbols.ts

```diff
@@ -132,9 +135,15 @@ export interface StoreOptions<T> {
   children?: any[];
 }
 
-export const enum LifecycleHooks {
-  NgxsOnInit = 'ngxsOnInit',
-  NgxsAfterBootstrap = 'ngxsAfterBootstrap'
+/**
+ * Represents a basic change from a previous to a new value for a single state instance.
+ * Passed as a value in a NgxsSimpleChanges object to the ngxsOnChanges hook.
+ */
+export class NgxsSimpleChange<T = any> {
+  constructor(public previousValue: T, public currentValue: T, public firstChange: boolean) {}
+  public isFirstChange(): boolean {
+    return this.firstChange;
+  }
```

Comment:
Does it need this method?
I feel like the `firstChange` property is good enough. What do you think?

Reply:
https://angular.io/api/core/SimpleChange#simplechange

Do we want to repeat this interface?

Reply:
Not necessarily, but we can take inspiration from it.
I think it is all good, but that method is just a bit strange to have there if you already expose a prop.

Reply:
done